translate/storage/resx.py

```diff
@@ -58,14 +58,16 @@ def source(self):
     def getsource(self):
         return self.source
 
-    def gettarget(self):
+    @property
+    def target(self):
         targetnode = self._gettargetnode()
         if targetnode is None:
             etree.SubElement(self.xmlelement, self.namespaced("value"))
             return None
         return data.forceunicode(targetnode.text) or u""
 
-    def settarget(self, target):
+    @target.setter
+    def target(self, target):
```

Comment:
As spotted in https://github.com/WeblateOrg/weblate/commit/c79304c2745c16661a13d1a900c09c621e87ab44#commitcomment-28783478, this breaks compatibility. Anybody who relied on settarget will not get proper behavior here as settarget from LISA will be called instead of customized behavior here.

Reply:
PR for this is here: https://github.com/translate/translate/pull/3806